.github/workflows/cd.yml

```diff
@@ -1,35 +1,28 @@
-# This workflow will build a Java project with Gradle
-# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
-
 name: CD
 
 on:
   workflow_dispatch:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-    env:
-      TEST_AUTH_TOKEN: ${{ secrets.ALPHA_TEST_AUTH_TOKEN }}
-      TEST_CACHE_NAME: dummy
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout project
+      uses: actions/checkout@v3
       with:
         token: ${{ secrets.MOMENTO_MACHINE_USER_GITHUB_TOKEN }}
-        submodules: true
+
     - name: Set up JDK 16
-      uses: actions/setup-java@v2
+      uses: actions/setup-java@v3
       with:
         java-version: 16
-        distribution: 'adopt'
-    - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
-    - name: Build with Gradle
-      run: ./gradlew clean build
-#    - name: Run integration tests
-#      run: ./gradlew integrationTest
+        distribution: 'coretto'
```

Comment:
Since AdoptOpenJdk is no longer developed I changed to coretto. I am open to any of them, however.